server/app.js

```diff
@@ -4,22 +4,75 @@ const cors = require('cors');
 const PORT = 3000;
 //const PORT = process.env.PORT || 3000;
 
-app.use(cors());
-
-const queries = [
-    { mitcnick: 'a coding guy' },
-    { students_mitcnick: 'guys/girls trying to code' },
-    { students_mitnick_lectures: 'a scary slack channel' },
-    { watercooler: 'memes' },
-]
-
+app.use(cors({
+  origin: '*'
+}));
+app.use(express.json())  // I don't really know what this does tbh
```

Comment:
I think this is adding the middleware Express to the app. So whenever a request hits your backend, Express will execute the functions you passed to app.use() in order.